flocker/provision/_install.py

```diff
@@ -517,48 +516,48 @@ def task_open_control_firewall(distribution):
 
 
 def task_enable_flocker_agent(distribution, control_node,
-                              dataset_backend=DatasetBackend.zfs,
-                              dataset_backend_configuration=dict(
-                                  pool=u'flocker'
-                              )):
+                              dataset_backend=None,
+                              dataset_backend_configuration=None):
     """
     Configure and enable the flocker agents.
```

Comment:
This only "Configure"s the agents in certain circumstances, sometimes it assumes that they are configured already. Update the docstring accordingly.